mmdet/models/detectors/base.py

```diff
@@ -273,7 +274,7 @@ def show_result(self,
                     bbox_color='green',
                     text_color='green',
                     thickness=1,
-                    font_scale=0.5,
+                    font_size=10,
```

Comment:
We should keep both the original argument and the new argument in case downstream codebases use this function. Also, add a deprecated warning for `font_scale`.